Dev-nkundineza/my-brand
# mmhI4Gmj-Ft-validate-login-contact-forms #1

# What does this PR do?

this PR is for Testing the feature of contact,login functions and validation. 
# Description of Task to be completed?

- [x] login validation
- [x] contact form validation
- [x] admin can logout
- [x] dashboard is protected
- [x] queries are displayed to the admin dashboard    

# How should this be manually tested?
clone the repo, run signin.html, the following are valid credentials for login(username(**admin**), password(**123**)), navigate to queries to check whether the queries inserted has been sent to admin.
# Any background context you want to provide?
# What are the relevant pivotal tracker stories?
https://trello.com/c/mmhI4Gmj/15-validate-login-and-contact-form-1
# Screenshots (if appropriate)
-[logout](https://user-images.githubusercontent.com/79087014/149897937-2ce9c44a-89b4-4135-b269-3be5f3191a87.JPG)
-[queries](https://user-images.githubusercontent.com/79087014/149898145-8a8d0c02-4981-4ba5-b557-319b17ceb3d6.JPG)




# Questions:
no question, rather I will appreciate your comments...
